src/commands/image/imageSource/containerRegistry/acr/createAcr/RegistryNameStep.ts

```diff
@@ -51,4 +52,29 @@ export class RegistryNameStep extends AzureWizardPromptStep<CreateAcrContext> {
             return { nameAvailable: true };
         }
     }
+
+    public static async tryGenerateRelatedName(context: DeployWorkspaceProjectContext, name: string): Promise<string | undefined> {
+        let registryAvailable: boolean = false;
+        let generatedName: string = '';
+
+        const timeoutSeconds: number = 60;
```

Comment:
Although rare that we would ever hit the full length of this timeout, I'm wondering if the timeout is too long to be hanging on an input name check.  Not really sure to be honest but just thought I'd bring it up, let me know what you think

Reply:
Yeah, I could see that. Maybe just try for 10 seconds?

Reply:
Yeah I personally think I'd be happy with anything between 10-30s.